lxd/instances_get.go

```diff
@@ -356,7 +360,7 @@ func doInstancesGet(d *Daemon, r *http.Request) (interface{}, error) {
 	// Get the data
 	wg := sync.WaitGroup{}
 	networkCert := d.endpoints.NetworkCert()
-	for address, instanceNames := range result {
+	for address, instancesAndProjects := range result {
```

Comment:
It should be `projectInstances` - project is above instance in the hierarchy, and comes first in the array.

Reply:
Project is currently second after instance in the array returned by `GetInstanceAndProjectNamesByNodeAddress`. Shall I change the returned array there?

Reply:
Oh right, yes please, projects should always come before instances (as a project can contain multiple instances), and an instance can only belong to a single project. Thanks